docs/diagrams/ui-sequence-diagram.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+activate ":Main" #FFBBBB
+":Main" -> ":UI": readUserInput()
+activate ":UI" #FFBBBB
+
+
+":UI" -> User: Gets user input
+activate "User" #FFBBBB
+User --> ":UI": "userInput: String"
```

Comment:
Maybe you can remove the quotation marks around userInput: String and for other instances in your seq diagram
![image](https://github.com/nus-cs2113-AY2324S1/tp/assets/93811746/5b3d00b1-f38f-4d92-b762-1b7e8dc1cc6f)